### PR TITLE
bsunanda:Run2-hcx20 Update the scenarios by adding relevant xml files

### PR DESCRIPTION
**Pull request by bsunanda**
Split some of the ECAL xml files to enable PhaseII definition and add files relevant for new ways of defining Hcal geometry

### CODE REVIEW

File: Geometry/CMSCommonData/python/cmsIdealGeometry2015XML_cfi.py [hunk 1]
```diff
@@ -217,8 +217,6 @@
         'Geometry/HcalCommonData/data/hcalouteralgo.xml',
         'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
         'Geometry/HcalCommonData/data/average/hcalforwardmaterial.xml',
-        'Geometry/HcalCommonData/data/Phase0/hcalSimNumbering.xml', 
-        'Geometry/HcalCommonData/data/Phase0/hcalRecNumbering.xml',
         'Geometry/MuonCommonData/data/v1/mbCommon.xml',
```
Comment (ianna) [inline Geometry/CMSCommonData/python/cmsIdealGeometry2015XML_cfi.py]: @bsunanda - why?

File: Geometry/CMSCommonData/python/cmsIdealGeometry2015devXML_cfi.py [hunk 2]
```diff
@@ -217,8 +217,6 @@
         'Geometry/HcalCommonData/data/hcalouteralgo.xml',
         'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
         'Geometry/HcalCommonData/data/average/hcalforwardmaterial.xml',
-        'Geometry/HcalCommonData/data/Phase0/hcalSimNumbering.xml', 
-        'Geometry/HcalCommonData/data/Phase0/hcalRecNumbering.xml',
```
Comment (ianna) [inline Geometry/CMSCommonData/python/cmsIdealGeometry2015devXML_cfi.py]: @bsunanda - please, correct.

**Comment by ianna** [comment]
@cmsbuild please test

**Comment by ianna** [comment]
+1

File: Geometry/CMSCommonData/python/Phase1_R34F16_cmsSimIdealGeometryXML_cfi.py [hunk 3]
```diff
@@ -232,7 +236,7 @@
         'Geometry/ForwardCommonData/data/totemRotations.xml', 
         'Geometry/ForwardCommonData/data/totemt1.xml', 
         'Geometry/ForwardCommonData/data/totemt2.xml', 
-        'Geometry/ForwardCommonData/data/ionpump.xml', 
+        'Geometry/ForwardCommonData/data/ionpump.xml')+cms.vstring(
```
Comment (davidlange6) [inline Geometry/CMSCommonData/python/Phase1_R34F16_cmsSimIdealGeometryXML_cfi.py]: @bsunanda - why the cms.vstring() is needed?

Comment (bsunanda) [inline Geometry/CMSCommonData/python/Phase1_R34F16_cmsSimIdealGeometryXML_cfi.py]: The list is too long
________________________________

**Comment by davidlange6** [comment]
+1

**Pull request merged**